arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -937,6 +937,7 @@
         "TTL": {
           "type": "integer",
           "format": "int64",
+	   "x-ms-client-name": "ttl",
```

Comment:
Nope, I didn't do it because of test failed, sorry.
Reworked it to eaisier case to make parameter name valid and equals to previous versions